terraform/modules/data-share-service/s3.tf

```diff
@@ -66,22 +66,3 @@ resource "aws_s3_bucket_acl" "cloudfront_logs_bucket_grants" {
     }
   }
 }
-
-module "ingress" {
-  source      = "../s3"
-  environment = var.environment
-  name        = "ingress"
-}
-
-module "ingress_archive" {
-  source      = "../s3"
-  environment = var.environment
-  name        = "ingress-archive"
-}
-
-module "egress" {
-  source          = "../s3"
-  environment     = var.environment
-  name            = "egress"
-  expiration_days = 7
-}
```

Comment:
I think the apply will fail here because these are not empty. We can either empty the buckets manually, or add a `force_destroy = true` here before removing them